### PR TITLE
Content deadlock fix with client <-> edge <-> core setup

### PR DESCRIPTION
**Pull request by aeplay**

### CODE REVIEW

**Review by marinoska — APPROVED** [review]

File: packages/cojson/src/sync.ts [hunk 1]
```diff
@@ -403,27 +403,39 @@ export class SyncManager {
     }
 
     if (entry.state.type === "loading") {
-      const value = await entry.getCoValue();
```
Comment (marinoska) [inline packages/cojson/src/sync.ts:406]: Probably just removing `await` keyword would be enough :)

Comment (aeplay) [reply packages/cojson/src/sync.ts:406]: ?? it wouldn't, everything in the then() still needs to happen after the covalue is loaded. (The diff view is very confusing unfortunately)

Comment (marinoska) [reply packages/cojson/src/sync.ts:406]: Oh sorry you're right!

**Review by gdorsi — APPROVED** [review]

**Comment by gdorsi** [comment]
The test is failing also on main, so don't worry about it

**Pull request merged**